internal/scheduler/gang_scheduler_test.go

```diff
@@ -513,9 +513,56 @@ func TestGangScheduler(t *testing.T) {
 				gangs = append(gangs, []*jobdb.Job{testfixtures.TestJob("A", jobId, "armada-preemptible-away-both", testfixtures.Test1Cpu4GiPodReqs("A", jobId, 30000))})
 				return
 			}(),
-			ExpectedScheduledIndices:       []int{1},
-			ExpectedScheduledJobs:          []int{0, 1},
-			ExpectedRuntimeGangCardinality: []int{0, 1},
+			ExpectedScheduledIndices:        []int{1},
+			ExpectedCumulativeScheduledJobs: []int{0, 1},
+			ExpectedRuntimeGangCardinality:  []int{0, 1},
+		},
+		"Home-away scheduling": {
```

Comment:
This tests that the priority is set correctly. Passes since adding the priority fix in nodedb.go.